lib/ui/screen-components/protected/restaurant(vendor)/dashboard/restaurant-stats-table/index.tsx

```
@@ -6,81 +6,102 @@ import { Divider } from 'primereact/divider';
+
+  return (
+    <div className="p-3 bg-gray-50 rounded-lg shadow-md space-y-6">
+      {paymentMethod.map((method: string) => {
+        const key: keyof typeof dashboardOrderSalesDetailsByPaymentMethod =
+          method as keyof typeof dashboardOrderSalesDetailsByPaymentMethod;
+
+        if (key !== 'all' && key !== 'cod' && key !== 'card') return;
+
+        return (
+          <>
+            <div className="flex flex-col space-y-2">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)